Vienna/Sources/Main window/BrowserView.m

```diff
@@ -72,7 +72,11 @@ -(void)configureTabClosingBehavior
 
 -(void)configureTabBar
 {
-	[self.tabBarControl setStyleNamed:@"Sierra"];
+    if (@available(macOS 10.10,*)) {
```

Comment:
Do we need the `@available` test (as the Mojave style can run under 10.9 in our branches) ?

Reply:
I used the @available test just to make sure there are no conflicts, as I did not test on a 10.9 VM. We could, however, lift it to 10.14 for having the Mojave style only in Mojave, or drop it to have it everywhere. I would recommend to do the first, since I also think that the Sierra style is more refined (but does not work with Mojave dark mode very well)

Reply:
I confirm that without the `@available` test, it works well in 10.9
The color may look a bit too contrasted to some people, but it's probably inevitable because there are always subtle tint differences between different OS versions.
Personally, I don't really care, as far as it is legible and functional.

Mojave style on 10.9
![capture d ecran 2018-10-19 a 01 53 19](https://user-images.githubusercontent.com/442030/47188965-f9613600-d342-11e8-964c-fcac09761513.png)

Sierra style on 10.9
![capture d ecran 2018-10-19 a 01 51 58](https://user-images.githubusercontent.com/442030/47188981-0e3dc980-d343-11e8-9d08-78c4785cbb20.png)

Reply:
Alright, then I will just remove the @available check.